internal/admin/dashboard/static/css/dashboard.css

```
@@ -774,6 +832,16 @@ body {
     font-size: 13px;
 }
 
+.col-price {
+    text-align: right;
+}
+
+td.col-price {
+    font-family: 'SF Mono', 'Menlo', 'Consolas', monospace;
```

Stylelint (17.3.0)

[error] Unexpected quotes around "Menlo" (font-family-name-quotes)

(font-family-name-quotes)